.mergify.yml

```diff
@@ -2,12 +2,13 @@ pull_request_rules:
   - name: backport to master
     conditions:
       - merged
-      - base=dev
+      - base=main
+      - milestone=1.13.x
```

Comment:
Do we want this `milestone` condition here? Or just make this conditional on the other items?